buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -22,8 +22,10 @@ object Dependencies {
 
     const val coroutinesCore =
         "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutinesCoreVersion"
+    const val coroutinesRx2 =
+        "org.jetbrains.kotlinx:kotlinx-coroutines-rx2:$coroutinesCoreVersion"
     const val coroutinesCoreTest = "org.jetbrains.kotlinx:kotlinx-coroutines-test:$coroutinesCoreVersion"
-    const val turbineCoroutinesTest = "app.cash.turbine:turbine:0.7.0"
+    const val turbineCoroutinesTest = "app.cash.turbine:turbine:0.8.0"
```

Comment:
BTW, this will update coroutines from 1.6.0 to 1.6.1 in tests. Can we update coroutines version explicitly for both tests and runtime?

Reply:
your call, don't have anything against that

Reply:
Done 😁